cmd/analyze/containerless_test.go

```
@@ -459,36 +463,18 @@ func TestAnalyzeCommandListTargetsContainerless(t *testing.T) {
-			// Create analyze command
-			a := &analyzeCommand{
-				AnalyzeCommandContext: AnalyzeCommandContext{
-					log:       log,
-					kantraDir: tmpKantraDir,
-				},
-			}
-
-			// Parse flags to set the command options
+			var listSources, listTargets bool
 			for _, arg := range tt.cmdArgs {
 				switch {
```

golangci-lint (2.12.2)

[error] QF1002: could use tagged switch on arg

(staticcheck)